README.md

```diff
@@ -1,46 +1,35 @@
 ## Vim Mode package [![Build Status](https://travis-ci.org/atom/vim-mode.svg?branch=master)](https://travis-ci.org/atom/vim-mode)
 
-Provides vim modal control for Atom, ideally blending the best of vim
-and Atom.
+Provides Vim modal control for Atom, blending the best of Vim and Atom.
 
-### Installing
+### Current Status
 
-Use the Atom package manager, which can be found in the Settings view or
-run `apm install vim-mode` from the command line.
+Sizable portions of Vim work as you'd expect, including many complex combinations.
+Even so, this package is far from finished (Vim wasn't built in a day).
 
-### Current Status
+If there's a feature of Vim you're missing, it might just be that you use it more often than other developers. Adding a feature can be quick and easy. Check out the [closed pull requests][prs] to see examples of community contributions, and look at the [issues][issues] to see if anyone has already solved your problem.
 
-Sizable portions of Vim's normal mode work as you'd expect, including
-many complex combinations. Even so, this package is far from finished (Vim
-wasn't built in a day).
+[prs]: https://github.com/atom/vim-mode/pulls?direction=desc&page=1&sort=created&state=closed
```

Comment:
Nit: This could probably be `&state=merged` instead to weed out the non-merged PRs.